### ISSUE TITLE
[BUG] Interact widgets not appearing in plot pane

### ISSUE THREAD

**Issue by yha** [JunoLab/Juno.jl#583]
After updating to 0.12.5, `Interact` output stopped showing in the plot pane

## Details
- Atom version: 1.47.0
- Julia version: 1.4.1
- OS: Windows 10
- Package versions:
  - Atom.jl: 0.12.11
  - julia-client: 0.12.5
  - ink: 0.12.4

### Steps to reproduce

1. Evaluate any `Interact` widget, e.g. `@manipualte` or
```
using Interact
slider(1:4)
```

The plot pane remains/becomes empty.

**Comment by yha** [comment]
The problem stays after downgrading `julia-client` to 0.12.4, `Juno` to 0.8.1, and `WebIO` to 0.8.13 (the versions I had before the last Pkg update). If I also downgrade `Atom` to 0.12.10, it doesn't precompile due to missing `Traceur` dependency.

Same problem?
JuliaGizmos/WebIO.jl#414

**Comment by aviatesk** [comment]
it seems like atom@1.47.0 disables webview (, which we use for displaying WebIO.jl views) by default.
[The fix](https://github.com/atom/atom/pull/20741) is already in master, but not sure when a release including that comes in.

Possible workaround would be to downgrade Atom to 1.46.0.

**Comment by yha** [comment]
> Possible workaround would be to downgrade Atom to 1.46.0.

This worked fine for me on one machine. On another, Julia doesn't start on Atom 1.46.0. Pressing "Press Enter to start Julia." leads to "Julia has exited.", and the error console has:
```
Julia exited with Error: The module '\\?\C:\Users\sternlab\.atom\packages\julia-client\node_modules\node-pty-prebuilt-multiarch\build\Release\conpty.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 70. This version of Node.js requires
NODE_MODULE_VERSION 69. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`)..

Uncaught (in promise) Error: The module '\\?\C:\Users\sternlab\.atom\packages\julia-client\node_modules\node-pty-prebuilt-multiarch\build\Release\conpty.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 70. This version of Node.js requires
NODE_MODULE_VERSION 69. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).
    at process.module.(anonymous function) [as dlopen] (ELECTRON_ASAR.js:160:31)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:722:18)
    at Object.module.(anonymous function) [as .node] (ELECTRON_ASAR.js:160:31)
    at Module.load (internal/modules/cjs/loader.js:602:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:541:12)
    at Function.Module._load (internal/modules/cjs/loader.js:533:3)
    at Module.require (file:///C:/Users/sternlab/Downloads/Atom-1.46.0/resources/app.asar/static/index.js:72:46)
    at require (C:\Users\sternlab\Downloads\Atom-1.46.0\resources\app\static\<embedded>:11:146745)
    at new WindowsPtyAgent (C:\Users\sternlab\.atom\packages\julia-client\node_modules\node-pty-prebuilt-multiarch\lib\windowsPtyAgent.js:35:36)
    at new WindowsTerminal (C:\Users\sternlab\.atom\packages\julia-client\node_modules\node-pty-prebuilt-multiarch\lib\windowsTerminal.js:49:24)
    at Object.spawn (C:\Users\sternlab\.atom\packages\julia-client\node_modules\node-pty-prebuilt-multiarch\lib\index.js:28:12)
    at file:///C:/Users/sternlab/.atom/packages/julia-client/lib/connection/process/basic.js:53:24

innerError Error: Cannot find module '../build/Debug/conpty.node'
    at Module._resolveFilename (internal/modules/cjs/loader.js:584:15)
    at Module._resolveFilename (C:\Users\sternlab\Downloads\Atom-1.46.0\resources\electron.asar\common\reset-search-paths.js:43:12)
    at Function.get_Module._resolveFilename (C:\Users\sternlab\Downloads\Atom-1.46.0\resources\app\static\<embedded>:11:155413)
    at Module.require (file:///C:/Users/sternlab/Downloads/Atom-1.46.0/resources/app.asar/static/index.js:61:43)
    at require (C:\Users\sternlab\Downloads\Atom-1.46.0\resources\app\static\<embedded>:11:146745)
    at new WindowsPtyAgent (C:\Users\sternlab\.atom\packages\julia-client\node_modules\node-pty-prebuilt-multiarch\lib\windowsPtyAgent.js:39:40)
    at new WindowsTerminal (C:\Users\sternlab\.atom\packages\julia-client\node_modules\node-pty-prebuilt-multiarch\lib\windowsTerminal.js:49:24)
    at Object.spawn (C:\Users\sternlab\.atom\packages\julia-client\node_modules\node-pty-prebuilt-multiarch\lib\index.js:28:12)
    at file:///C:/Users/sternlab/.atom/packages/julia-client/lib/connection/process/basic.js:53:24
```
This problem persists with reinstalling or downgrading `julia-client` and `ink`.

**Comment by aviatesk** [comment]
another update on this:
I confirmed [atom≧v1.49.0-nightly](https://github.com/atom/atom-nightly-releases/releases/tag/v1.49.0-nightly4) addresses this issue.

**Comment by bchain** [comment]
I tried Atom  1.48.0-beta in Ubuntu, and it opened a plot window but did not show anything.

**Comment by aviatesk** [comment]
try nightly

**Comment by bchain** [comment]
Works with nightly !  Will this filter into the next stable version automatically ?

**Comment by aviatesk** [comment]
don't know, this is all upstream (i.e. atom itself) issue, so we can't do anything other than waiting it's released into stable.

**Comment by yha** [comment]
Looks like the fix is included in atom 1.48.0, released 4 days ago.
Interact widgets work fine for me now.
